src/styles/styles.scss

```diff
@@ -5,7 +5,7 @@
 			fill: var(--color-warning);
 		}
 		&--white {
-			fill: var(--color-primary-text);
+			fill: var(--color-primary-element-text);
```

Comment:
Those 2 colors can never diverge, both are either black or white and that does not change by the "-element" cut of :P

Reply:
I see. Then it is probably fine. However I wonder if we should still normalize this here as well.

Reply:
IIRC the problem goes even further because it still assumes it would be on primary color from the topbar, but it is on the background image, so the white might not be visible properly and instead of assigning a color we need to fall back to the normal font color, but as said. needs more testing to have clear results for all colors and states at the end.

Reply:
I see